src/v2/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -16,7 +16,29 @@ const AuctionsApp: React.FC<AuctionsAppProps> = ({ children }) => {
   return (
     <AppContainer>
       <AuctionsMeta />
-
+      <Box mt={3} ml={4}>
+        <Text variant="largeTitle">Auctions</Text>
+        <Text pt="15px" pb="15px">
```

Comment:
When using spacing props (margin, padding, etc) we'll always want to stick with the values that are [defined in our theme](https://github.com/artsy/palette/blob/master/packages/palette/src/Theme.tsx#L168-L194) so that we don't end up with design drift. 

(Design drift is where, at scale, one sees things like `9px` here, `11px` there, `4px` somewhere else, leading to subtly inconsistant layouts that don't look quite right.) 

By constraining things to the spacing scale our UI ends up like legos where everything looks internally consistent.

All of this is to say, when you see a figma comp that has a margin or padding that doesn't quite match one of the values in the theme (which is generally round numbers) just round up or down to the closest thing, using your best jugement. So here it could be either a `2` or a `1`:

```
<Text pt={2} pb={2}>
```

(meaning, 20px or 10px)

Even better tho, you can collapse these two props into just one, `py` -- "padding y", which applies it to both top and bottom. The same goes for horizontal values: `px` (or even `mx`, if you need a margin).

Reply:
Added this in, thanks! Could not find this in palette.